### ISSUE TITLE
[BITV] 9.4.1.2/8.5 - The "Settings" button is an accordion button that should be implemented as such with the aria-expanded attribute. 

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/server#37136]
See https://www.w3.org/WAI/ARIA/apg/patterns/accordion/examples/accordion/ for further details. (1)

- [ ] Fix
- [ ] Backport

![4b98244143224a248f230b7ff75c24c8](https://user-images.githubusercontent.com/1315170/223752885-7c189fc6-06e1-4cee-a850-409df648dec4.png)

<details>
<summary>Details</summary>

https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-bfbf07e5c3-v8-n5

</details>
